src/GovUk.Education.ExploreEducationStatistics.Data.Api.Tests/Controllers/TableBuilderControllerTests.cs

```diff
@@ -57,25 +56,25 @@ public class TableBuilderControllerTests(TestApplicationFactory testApp)
                 .WithQuery(new ObservationQueryContext
                 {
                     SubjectId = Guid.NewGuid(),
-                    Filters = new List<Guid>(),
-                    Indicators = new List<Guid>(),
-                    LocationIds = new List<Guid>(),
+                    LocationIds = [ Guid.NewGuid(), ],
                     TimePeriod = new TimePeriodQuery
                     {
                         StartYear = 2021,
                         StartCode = CalendarYear,
                         EndYear = 2022,
                         EndCode = CalendarYear
-                    }
+                    },
+                    Filters = new List<Guid>(),
+                    Indicators = new List<Guid> // use collection expression -> test failures
```

Comment:
Not sure on the meaning/purpose of this comment - leftover note-to-self?

Reply:
It's just a note that if you switch this from `new List<Guid>` to a collection expression, the unit test fails. I'll change the comment to make it clearer.